modules/calib3d/src/calibinit.cpp

```diff
@@ -471,6 +471,110 @@ static void icvBinarizationHistogramBased(Mat & img)
     }
 }
 
+static bool findCornerNeighbor(
+    cv::AutoBuffer<ChessBoardQuad>& all_quads,
+    const std::vector<Point2f>& all_quads_pts,
+    flann::GenericIndex<flann::L2_Simple<float>>& all_quads_pts_index,
+    const int idx,
+    const ChessBoardQuad& cur_quad,
```

Comment:
`cur_quad` is `all_quads[idx]`, I do not think that it's needed in the function parameters.

Reply:
Done